src/modules/item/utils.spec.ts

```diff
@@ -50,3 +50,18 @@ describe('when transforming third party items to be sent to a contract method',
     ])
   })
 })
+
+describe('when building building zip files and representations are equal', () => {
```

Comment:
There's a repeated `building` here haha.
```suggestion
describe('when building zip files and representations are equal', () => {
```

Reply:
fixed :)